libraries/AP_HAL_Linux/HAL_Linux_Class.cpp

```diff
@@ -276,6 +277,12 @@ void _usage(void)
     printf("\tmodule support:\n");
     printf("\t                   --module-directory %s\n", AP_MODULE_DEFAULT_DIRECTORY);
     printf("\t                   -M %s\n", AP_MODULE_DEFAULT_DIRECTORY);
+#ifdef MQTT_ENABLED
+    printf("\tmqtt options:\n");
+    printf("\t                   --mqtt_server tcp:\\192.168.1.x:xxxx\n");
```

Comment:
The IP adress is not fixed, it just an example. According to the mqtt library, the url should takes the following form:
<protocol>:\\<host>:<port>